src/components/PaymentMethodForm/PaymentMethodForm.tsx

```diff
@@ -63,8 +63,9 @@ const PaymentMethodForm = ({
         }));
         return listItems || [];
     }, [availablePaymentMethods]);
+
     const handleGoBack = () => {
-        if (isDirty) {
+        if (Object.keys(dirtyFields).length) {
```

Comment:
`isDirty` doesn't work?

Reply:
nope, for some reason, isDirty is always true, so another way to check it is through dirtyFields, cause at the beginning it will always be an empty object